py/server/deephaven/completer/__init__.py

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
If this needs to be public, I would prefer it is called `autocomplete`, since that is more clear to the user looking at the package.